### PR TITLE
alpscore : update to 0.5

### PR DESCRIPTION
**Pull request by aeantipov**
The ALPSCore project, based on the ALPS (Algorithms and Libraries for Physics Simulations) project, provides generic algorithms and utilities for physics problems. It strives to increase software reuse in the physics community.

This release: Bugfixes and enhancements. Some of the visible changes are:

1. alps::accumulators changes:
     - improved access to individual accumulators (not as part of accumulator_set)
     - Allowed vector-constant and vector-scalar binary operations.
2. alps::params changes:
     - Specifications clarified. In particular, a parameter containing a "larger" type can be assigned a value of a "smaller" type, but cannot be converted to the value of a "smaller" type.
     - Improved adherence to the specifications.
     - Comparison operators are added for convenience.
     - API changes: defaulted(), defined(), exists().

For copyright see COPYRIGHT.txt. For licensing see LICENSE.txt. For acknowledgment in scientific publications see ACKNOWLEDGE.txt.

PS Previous pull request was incorrect (bottle links explicitly removed). The commit was removed and redone to follow the "homebrew-science/Contributing" guide.

### CODE REVIEW

**Comment by dpo** [comment]
@BrewTestBot test this please

File: alpscore.rb [hunk 1]
```diff
@@ -1,7 +1,8 @@
 class Alpscore < Formula
+  desc "Applications and Libraries for Physics Simulations"
   homepage "http://alpscore.org"
-  url "https://github.com/ALPSCore/ALPSCore/archive/v0.4.5.tar.gz"
-  sha256 "b043f5043f6fdca5efd8e1fc2ba0d893da0fd04bff8adaa213c797b44d68e72e"
+  url "https://github.com/ALPSCore/ALPSCore/archive/v0.5.tar.gz"
+  sha256 "05fa4bc65eba8f2e09ef89907ed5763aa775289654ea324a25d403836a1cbb28"
   head "https://github.com/ALPSCore/ALPSCore.git"
   revision 1
```
Comment (dpo) [inline alpscore.rb]: Please remove the revision when upgrading to a new version.

**Comment by dpo** [comment]
Please also squash your commits. Thanks!

**Comment by aeantipov** [comment]
Thanks, I did it now.

**Comment by dpo** [comment]
Thank you!

**Pull request closed without merging**